pkgs/by-name/sq/sqruff/package.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, stdenvNoCC
+, fetchurl
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "sqruff";
+  version = "0.11.0";
+  src = fetchurl {
+    url = "https://github.com/quarylabs/sqruff/releases/download/v0.11.0/sqruff-linux-x86_64-musl.tar.gz";
```

Comment:
```suggestion
stdenvNoCC.mkDerivation rec {
  pname = "sqruff";
  version = "0.11.0";
  src = fetchurl {
    url = "https://github.com/quarylabs/sqruff/releases/download/v${version}/sqruff-linux-x86_64-musl.tar.gz";
```

Reply:
Alternatively,
```suggestion
let
  pname = "sqruff";
  version = "0.11.0";
in
stdenvNoCC.mkDerivation {
  inherit pname;
  inherit version;
  src = fetchurl {
    url = "https://github.com/quarylabs/sqruff/releases/download/v${version}/sqruff-linux-x86_64-musl.tar.gz";
```

Reply:
I could do this with `stdenvNoCC.mkDerivation (finalAttrs: {` instead of `rec` iydm?

Reply:
I did that